test/e2e_node/eviction_test.go

```diff
@@ -395,6 +395,26 @@ var _ = SIGDescribe("PriorityMemoryEvictionOrdering", framework.WithSlow(), fram
 			{
 				evictionPriority: 1,
 				pod:              getMemhogPod("high-priority-memory-hog-pod", "high-priority-memory-hog", v1.ResourceRequirements{}),
+				prePodCreationModificationFunc: func(pod *v1.Pod) {
+					nodeList, err := f.ClientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+					framework.ExpectNoError(err)
+					gomega.Expect(nodeList.Items).To(gomega.HaveLen(1))
```

Comment:
this reminds me of `getLocalNode`. Question: is `getLocalNode` fit to be used here? if not it's fine, but let's add a one-liner comment explaining why.